controllers/users/users.js

```diff
@@ -4,6 +4,43 @@ const jwt = require("jsonwebtoken");
 const config = require("config");
 const User = require("../../models/User");
 const messages = require("../messages");
+const nodemailer = require("nodemailer");
+require("dotenv").config();
+
+const sendVerificationEmail = user => {
+  const transporter = nodemailer.createTransport({
+    service: "gmail",
+    auth: {
+      user: process.env.GMAIL_USER,
+      pass: process.env.GMAIL_PASS,
+    }
+  })
+
+  const mailOptions = {
+    from: "natripareact <natripareact@gmail.com>",
+    to: user.email,
+    subject: "Account Activation",
+    html: `
+      To activate your account, click this link
+      <a href="http://localhost:5000/api/users/activate/${user.activationKey}">Activate Account</a>
+    `
+  }
+
+  transporter.sendMail(mailOptions, (err, data) => {
+    if(err) throw err;
+  })
+}
+
+exports.activateUser = async (req, res) => {
+  const { activationKey } = req.params;
+  await User.findOneAndUpdate({ activationKey }, { emailVerified: true })
```

Comment:
What should be returned if activation key is expired not exists or already used?